internal/Picker/Picker.js

```diff
@@ -245,8 +261,10 @@ const PickerBase = class extends ReactComponent {
 
 		/**
 		 * Determines the user interaction of the control. A joined picker allows the user to use
-		 * the arrow keys to adjust the picker's value. The user may no longer use those arrow keys
-		 * to navigate, while this control is focused. A split control allows full navigation,
+		 * the arrow keys or the enter key to adjust the picker's value.
+		 * Whether to use the arrow keys or the enter key depens on
+		 * the value of the [changedBy]{@link sandstone/internal/Picker.PickerBase.changedBy}.
```

Comment:
I think the original one is better.
```suggestion
		 * It depends on [changedBy]{@link sandstone/internal/Picker.PickerBase.changedBy}
		 * whether to use the arrow keys or the enter key.
```